Main.py

```diff
@@ -9,5 +9,16 @@
 def testCall():
     print("testCall()")
 
+def getRandomString(length):
+    """Generate a random string of fixed length """
+    letters = string.ascii_letters
+    return ''.join(random.choice(letters) for i in range(length))
+
+def getRandomInt(min, max):
+    """Generate a random integer between min and max"""
+    return random.randint(min, max)
+
 if __name__ == "__main__":
-    testCall()
\ No newline at end of file
+    testCall()
+    print(getRandomString(10))
```

Comment:
Now you dont need to print the function here, just the function instead
```suggestion
    getRandomString(10)
```

Reply:
Taken care